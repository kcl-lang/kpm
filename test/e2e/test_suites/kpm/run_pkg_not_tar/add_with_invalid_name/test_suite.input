kpm add not_exist