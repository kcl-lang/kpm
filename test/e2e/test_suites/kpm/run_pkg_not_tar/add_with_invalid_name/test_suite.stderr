kpm: failed to select latest version from 'localhost:5001/test/not_exist'.
